hyperspy/_signals/lazy.py

```diff
@@ -131,6 +133,29 @@ def compute(self, close_file=False, show_progressbar=None, **kwargs):
 
     compute.__doc__ %= SHOW_PROGRESSBAR_ARG
 
+    def rechunk(self,
+                nav_chunks="auto",
+                sig_chunks=-1,
+                **kwargs):
+        """Rechunks the data using the same rechunking formula from Dask expect
+        that the navigation and signal chunks are defined seperately. Note, for most
+        functions sig_chunks should remain None so that it spans the entire signal.
+
+        Parameters:
+        nav_chunks:
+            The navigation chunk size
+        sig_chunks:
+            The signal chunk size
+        """
+        if not isinstance(sig_chunks, tuple):
```

Comment:
> Ideally, this function should be used in `_lazy_data`, so that it will be easier to maintain and ensure that the rechunking is performed in a consistent manner but we can leave it for another PR.

One for #2669.